app/views/layouts/login.html.haml

```diff
@@ -1,4 +1,4 @@
-!!!
+= render :partial => 'layouts/doctype'
```

Comment:
Exactly, that is the reason. Have just one and make it very explicit that we have one.